tests/e2e/tools/lib/editor.js

```diff
@@ -64,10 +66,36 @@ async function removeAllPostTypes({editor}) {
   }
 }
 
+/**
+ * Insert new block into page using the block inserter
+ *
+ * @param {Object} page             - The page object used to interact with the page.
+ * @param {Object} params           - Additional block's parameters.
+ * @param {Object} params.blockName - The name of the block.
+ * @param {Object} params.namespace - The namespace to search if it is needed.
+ * @return {Promise<void>}          - Playwright Locator
+ */
+const searchAndInsertBlock = async (page, {blockName, namespace = ''}) => {
+  // Can be also used the aria-label="Toggle block inserter"
```

Comment:
As mentioned in this [comment](https://github.com/greenpeace/planet4-master-theme/pull/2419#discussion_r1810824342) when needing to add several blocks in a row for a test, it seems that the `Toggle block inserter` version is the only one that works so we should maybe use it instead?